RELEASE_PROCESS.md

```diff
@@ -78,9 +84,29 @@ Building a Compose release
 
         git push git@github.com:docker/compose.git $TAG
 
-8.  Create a release from the tag on GitHub.
+8.  Draft a release from the tag on GitHub.
+
+    - Go to https://github.com/docker/compose/releases and click "Draft a new release".
+    - In the "Tag version" dropdown, select the tag you just pushed.
+
```

Comment:
@moxiegirl It's not sub-procedure, it's copy for the release notes which is meant to be copy/pasted into the GitHub release. What would be the best way to make that clear?